data/README.md

```diff
@@ -15,7 +15,8 @@ En cas de problème vous pouvez redémarrer de zéro en faisant d'abord un `make
 ## Autres commandes :
 
 * `make image` : pour construire l'image docker choisie
-* `make import_agribalyse` : pour importer la base dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
+* `make import_agribalyse` : pour importer Agribalyse 3.0 dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
```

Comment:
Trying this command, I got these warnings:

```
WARNING: The directory '/home/jovyan/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
```

and 

```
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```

I'm not too worried about the last one but the first might be source of a performance penalty.

Reply:
Also, I'm getting this error:

```
FileNotFoundError: [Errno 2] No such file or directory: 'AGB3.1.1.20230306.CSV.zip'
```

… while the docs mention `agribalyse3_no_param.CSV.zip`. At this point I'm confused and blocked.

Reply:
Thanks, I've updated the doc and a few commands. The needed files are on the jupyter server, some of them are still private (EI 3.9.1).   If you send your pub key, I can install it so that you can try the full process, at least for food:
```
cd ecobalyse/data
scp ecobalyse@jupyter.ecobalyse.fr:ecobalyse/data/AGB3.1.1.20230306.CSV.zip .
scp ecobalyse@jupyter.ecobalyse.fr:ecobalyse/data/'Environmental Footprint 3.1 (adapted).CSV' .
make clean image import_agribalyse import_method export_builder
```

Reply:
My pubkey:

```
ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAxVjMS7v/JBi6E4TS7/fafInrqtqmWGs9ivB2OXpvpdT16SrXlu3/L3HrUPESZBtm5a7Xnddp8DE5Jp4qJIUBUsER3nNPEG9daTxpYSrfK/rZPIx1Nj1aDrpTzncOA5qhXy/lHYv1f7dUmB3zLVf8xmg/06R16Zs2JH2cSQdrGaCIX0pi9aGeimh0EMHmQ1dYLWzHYUMNh1ycQRSBOYTH0wmpBxFXRXtlkwAuhNyZzBCwx78A1PTmRIsgRQQadW+EYA3YQY1SlLkDr++uf8tXL37v4tUBrgIluMGepN4a7a44yCsDd0/4YMkViKAszn7lIB5vU6WkfXzuhMJqrg/R3w==
```

Reply:
So I could test the whole setup and export now I can retrieve the appropriate ZIP files, and it works smoothly. I think we should better document what these ZIP files are and what is the exact procedure to get them, otherwise people may argue the product is not fully opensource.